src/assets/lima-config.yaml

```diff
@@ -42,12 +42,32 @@ provision:
   mode: system
   script: |
     #!/bin/sh
+    set -o errexit -o nounset -o xtrace
     if ! [ -d /mnt/data/root ]; then
       mkdir -p /root
       mv /root /mnt/data/root
     fi
     mkdir -p /root
     mount --bind /mnt/data/root /root
+- # Create /etc/docker/certs.d symlink
+  mode: system
+  script: |
+    #!/bin/sh
+    set -o errexit -o nounset -o xtrace
+    mkdir -p /etc/docker
+
+    # Delete certs.d if it is a symlink (from previous boot).
+    [ -L /etc/docker/certs.d ] && rm /etc/docker/certs.d
+
+    # Create symlink if certs.d doesn't exist (user may have created a regular directory).
+    if [ ! -e /etc/docker/certs.d ]; then
+      # We don't know if the host is Linux or macOS, so we take a guess based on which mountpoint exists.
```

Comment:
Can't we actually look at the mounts to figure out which we want? (That is, read through `lima.env`)

Reply:
How does that help? There can be additional mounts at the front and end of the list, from `override.yaml` and `default.yaml`. So how do we know which one is the home directory?

The directories are also not yet mounted when the provision scripts execute (unless you use 9p), so the only thing we can rely on that `mkdir -p $MOUNTPOINT` has been executed for each mount.

Reply:
That sounds like a "no", then :)